### DISCUSSION TITLE
OctoLamp build - Bambu Lab X1 Carbon by Matthijs

### DISCUSSION THREAD

**Discussion by MatthijsvdVeer** [martinwoodward/octolamp#4] [Show and tell]
![image](https://user-images.githubusercontent.com/736727/213911391-62d7fdce-9b43-47f8-83da-1cb8ce3b3ee7.png)

Happy with this build. This project is accessible to all levels of 3D printing. I did run into a tiny problem with the white inner part. There is a nice rounded edge on the outside ring, but on the inside, there isn't. My printer has very strong bed adhesion, so the inner ring split from the flat part when removing it from the bed. Nothing that can't be fixed with a bit of glue, but having that same rounded edge on the inside would make it even better. Here's the area that would benefit from it:
<img width="823" alt="image" src="https://user-images.githubusercontent.com/736727/213911549-b7c49894-e276-4198-a173-b9281a748345.png">

Thanks a lot for creating this project

**Comment by martinwoodward** [comment]
Good point, thanks. Will take a look, I was having some issues with Fusion when attempting to chamfer the inside. Will take another look.

Great looking build - thanks for sharing
